### PR TITLE
Split operator raft command into subcommands

### PR DESCRIPTION
**Pull request by kyhavlov**
This changes the operator command to match the `kv` and `snapshot` commands in being organized into subcommands, while leaving the old flags hidden (until 0.9) for compatibility.

### CODE REVIEW

File: command/operator_raft_list.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error getting peers: %v", err))
+	}
+	c.Ui.Output(result)
+
+	return 0
+}
+
+func raftListPeers(operator *api.Operator, stale bool) (string, error) {
+	q := &api.QueryOptions{
```
Comment (slackpad) [inline command/operator_raft_list.go]: Do we need this with the new command stuff?

File: command/operator_raft_remove.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+type OperatorRaftRemoveCommand struct {
+	base.Command
+}
+
+func (c *OperatorRaftRemoveCommand) Help() string {
+	helpText := `
+Usage: consul operator raft remove-peer [options]
+
+Remove the Consul server with given -peer-address from the Raft configuration.
```
Comment (slackpad) [inline command/operator_raft_remove.go]: This looks like it was wrong before - should just be `-address`.

File: website/source/layouts/docs.erb [hunk 3]
```diff
@@ -141,6 +141,19 @@
 
 					<li<%= sidebar_current("docs-commands-operator") %>>
 					<a href="/docs/commands/operator.html">operator</a>
+					<ul class="subnav">
```
Comment (slackpad) [inline website/source/layouts/docs.erb]: I think for now let's just show the subcommands (just raft for now).

File: command/operator_raft.go [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+	f.BoolVar(&removePeer, "remove-peer", false,
+		"If this flag is provided, the Consul server with the given -address will be "+
+			"removed from the Raft configuration.")
+
+	// Parse other arguments.
+	var address string
+	f.StringVar(&address, "address", "",
+		"The address to remove from the Raft configuration.")
+
+	c.Command.HideFlags("list-peers", "remove-peer", "address")
```
Comment (slackpad) [inline command/operator_raft.go]: Let's put a comment w/a TODO about why these are here.

File: command/operator_raft_list.go (hunk 1 above)
Comment (kyhavlov) [inline command/operator_raft_list.go]: the api client doesn't have any way to set a config setting for stale when initializing it, so this is the only way currently

File: website/source/layouts/docs.erb (hunk 3 above)
Comment (kyhavlov) [inline website/source/layouts/docs.erb]: Move the bit on the raft subcommand page to the operator one, and show `raft list-peers` and `raft remove-peer` instead for the sidebar?

Comment (slackpad) [inline website/source/layouts/docs.erb]: I'd say get rid of `list-peers` and `remove-peers` so it looks like this:

* Consul Commands (CLI)
    * operator
        * raft

File: command/operator_raft_list.go (hunk 1 above)
Comment (slackpad) [inline command/operator_raft_list.go]: Ah ok.

**Pull request merged**